Riot/Managers/Theme/Themes/DarkTheme.swift

```diff
@@ -130,32 +130,26 @@ class DarkTheme: NSObject, Theme {
             let appearance = UINavigationBarAppearance()
             
             if BuildSettings.newAppLayoutEnabled {
-                appearance.configureWithDefaultBackground()
+                appearance.configureWithTransparentBackground()
+                appearance.backgroundEffect = UIBlurEffect(style: .dark)
```

Comment:
This changes the navigation bar material so it is now non-standard.
| Before | After |
| - | - |
| ![Simulator Screen Shot - iPhone 13 mini - 2022-08-04 at 15 10 45](https://user-images.githubusercontent.com/6060466/182869294-3976f675-a320-4496-88e9-95304ccf254c.png) | ![Simulator Screen Shot - iPhone 13 mini - 2022-08-04 at 15 14 05](https://user-images.githubusercontent.com/6060466/182869353-d81dcc37-8344-4040-973e-5bec60afe64b.png) |

Reply:
I opted out for the opaque bar